### PR TITLE
fix: add mention `.lens` suffix to send publication to server

### LINKED ISSUE #1342: Remove `.lens` everywhere
### Summary

- [ ] <Slug /> component
- [ ] URLs
- [ ] Mentions
- [ ] Mention in composer

But still we should not touch the service layer, all needs to be done in UI layer

### PR DESCRIPTION
**Pull request by hungds99**
## What does this PR do?

Fixes #1342

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist

- I haven't read the [contributing guide](https://github.com/lensterxyz/lenster/blob/main/CONTRIBUTING.md)
- My code doesn't follow the style guidelines of this project
- I haven't performed a self-review of my own code and corrected any misspellings
- I haven't commented my code, particularly in hard-to-understand areas
- I haven't checked if my PR needs changes to the documentation
- I haven't checked if my changes generate no new warnings
- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes

### CODE REVIEW

File: apps/web/package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "build": "next build",
     "start": "next start",
     "typecheck": "tsc --noEmit",
-    "lint": "eslint . --ext .js,.ts,.tsx"
+    "lint": "eslint src/ --ext .js,.ts,.tsx"
```
Comment (hungds99) [inline apps/web/package.json:10]: @bigint I have an issue when run Eslint not response

Comment (bigint) [reply apps/web/package.json]: Hope this is resolved 🤞🏼

File: packages/data/constants.ts [hunk 2]
```diff
@@ -85,6 +85,7 @@ export const ADDRESS_REGEX = /^(0x)?[\da-f]{40}$/i;
 export const HANDLE_REGEX = /^[\da-z]+$/;
 export const ALL_HANDLES_REGEX = /([\s+])@(\S+)/g;
 export const HANDLE_SANITIZE_REGEX = /[^\d .A-Za-z]/g;
+export const MENTION_REGEX = /@[\w.-]+/;
```
Comment (bigint) [inline packages/data/constants.ts:88]: can we name it very brief, we have lot of confusing names above 🙏🏼

**Comment by bigint** [comment]
Oops @hungds99 🙇🏼

**Comment by hungds99** [comment]
@bigint I deleted my forked repos =)). Do you have any feedback ?

**Pull request closed without merging**